components/brave_wallet_ui/common/async/lib.ts

```diff
@@ -1108,3 +1109,20 @@ export async function isTokenPinningSupported (token: BraveWallet.BlockchainToke
   const { braveWalletPinService } = getAPIProxy()
   return await braveWalletPinService.isTokenSupported(token)
 }
+
+
+export function refreshPortfolioFilterOptions () {
+  return async (dispatch: Dispatch, getState: () => State) => {
+    const { wallet: { accounts, networkList, selectedAccountFilter, selectedNetworkFilter } } = getState()
+
+    if (![...networkList, AllNetworksOption].some(network => network.chainId === selectedNetworkFilter.chainId)) {
```

Comment:
might be faster without spreading networks to a new array
```suggestion
    if (!networkList.some(network => network.chainId === selectedNetworkFilter.chainId) && selectedNetworkFilter.chainId !==  AllNetworksOption.chainId) {
```

Reply:
Done!